website/docusaurus.config.js

```diff
@@ -119,8 +119,8 @@ module.exports = {
         prism: {
             additionalLanguages: ['rust', 'toml'],
         },
-        googleAnalytics: {
-            trackingID: 'UA-141789564-1',
+        gtag: {
+            trackingID: 'G-DENCL8P4YP',
```

Comment:
I am not very familiar with the tracking we use, but is this change intentional?

Reply:
Yes, previous one was broken. I realized that while creating a new version